### DISCUSSION TITLE
Change Https and http for routerfleet

### DISCUSSION THREAD

**Discussion by Anigbani** [eduardogsilva/routerfleet#48] [General]
Hello,

I'm new to linux and docker and i was wondering if i can change the 443 and 80 port for my routerfleet instance since i have a linux server with multiple instance installed and the 443 is been taken ? 

Thank you in advance,

**Comment by matteocisilino** [comment]
Hello Anigbani , 
you can do by apply the modification in your  configuration files accordingly 

1. docker-compose.yml ports part
2. .env SERVER_ADDRESS

1. https://docs.docker.com/get-started/docker-concepts/running-containers/publishing-ports/
2. ports are passed to the proxy as ADDRESS part

what i will write after this line assume you have basic knowledge on YAML 

Let's assume you want to use 8443 for HTTPS and 8080 for HTTP 
changes in docker-compose.yml will be : 

```
ports:
- "8080:80"
- "8443:443"
``` 
after that you will have a CSRF problem because the proxy don't parse the ports , you must change also the .env file 

`SERVER_ADDRESS=HOSTNAME:8443`
